src/calendar/interfaces.ts

```diff
@@ -78,6 +78,12 @@ export interface CalendarProps extends BaseComponentProps {
    * @i18n
    */
   i18nStrings?: CalendarProps.I18nStrings;
+
+  /**
+   * Specifies the granularity at which users will be able to select a date.
+   * Defaults to `day`.
+   **/
+  granularity?: 'month' | 'day';
```

Comment:
Shouldn't the type be `CalendarProps.Granularity` ?

Reply:
Yes, updated.